delay.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#Program to simulate network delay between nodes
+# Run as follows: "./tc_new.sh <instance_number>
+
+
+#Create IP Delay Tables
+IP_list=( 128.111.84.146 128.111.84.157 128.111.84.160 128.111.84.161 128.111.84.165 )
+
+let "instance_no = $1 - 1"
+
+declare -A DTable
+declare -a Delays
+Delays=(1 22 65 136 185 22 1 88 125 182 65 88 1 73 121 136 125 73 1 185 185 182 121 185 1)
```

Comment:
Are these random or between actual AWS datacenters?

Reply:
These are the actual delays between AWS datacenters in California, Oregon, Virginia, Ireland and Sao Paolo.